src-tool/Pact/Analyze/Types.hs

```diff
@@ -127,3 +131,14 @@ data ModuleRefs = ModuleRefs
   } deriving Show
 
 makeLenses ''ModuleRefs
+
+data VerificationWarning
+  = FVDuplicatedPropertyDef [Text]
```

Comment:
In the future it might be nice to replace the bare constructors with record constructors, so that you can name the `[Text]` and `Text Text` arguments, like

```
data VerificationWarning
  = FVDuplicatedPropertyDef { overloadedNames: [Text] }
  | FVShimmedStaticContent { content: Text, shim: Text }
```

The record accessors are not safe to use (they're partial), but record destructuring in a pattern match makes this type friendly to consume.

Reply:
Alternatively, just adding haddock comments describing the parameters.